src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/RangeFilterClassTest.java

```diff
@@ -1,45 +1,75 @@
 package dk.sdu.se_f22.sortingmodule.range.rangepublic;
 
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.parallel.Execution;
-import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-//@Execution(ExecutionMode.CONCURRENT)
-// Remember to check whether concurrent has a detrimental impact on performance
 class RangeFilterClassTest {
-    @Test
-    @Disabled("not written yet")
-    void testEquals() {
-        // This test should be split up into many tests to test each step/level of the equals method
+    @Nested
+    @DisplayName("Non-matching dummy filter equals test")
```

Comment:
This has the same naming concerns as doublefilter equals

Reply:
Done